src/useDCPWorker.tsx

```diff
@@ -355,249 +291,265 @@ function getWorkerOptionsKey() {
  *
  *  @returns {WorkerOptions?} The worker options, or null if they weren't found.
  */
-function loadWorkerOptions() {
-  let options = window.localStorage.getItem('worker-options');
-  let storage = options === null ? {} : JSON.parse(options);
+function loadWorkerOptions()
+{
+  const options = window.localStorage.getItem('dcp-worker-options');
+  const storage = options === null ? {} : JSON.parse(options);
 
   let loadedOptions;
 
-  if (Object.prototype.hasOwnProperty.call(storage, getWorkerOptionsKey())) {
+  if (Object.prototype.hasOwnProperty.call(storage, getWorkerOptionsKey()))
+  {
     loadedOptions = storage[getWorkerOptionsKey()];
-  } else if (
-    Object.prototype.hasOwnProperty.call(storage, 'defaultMaxWorkers')
-  ) {
+  }
+  else if (Object.prototype.hasOwnProperty.call(storage, 'defaultMaxWorkers'))
+  {
     loadedOptions = storage;
   }
 
-  if (!loadedOptions) return null;
+  if (!loadedOptions)
+    return null;
 
-  if (
-    Object.prototype.hasOwnProperty.call(loadedOptions, 'paymentAddress')
-  ) {
-    if (loadedOptions.paymentAddress instanceof window.dcp.wallet.Keystore) {
+  if (Object.prototype.hasOwnProperty.call(loadedOptions, 'paymentAddress'))
```

Comment:
**suggestion:** You may want to look into [`Object.hasOwn`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwn) to cut down on the verbosity (added 2021/2022).

**praise:** Kudos on not falling back to simply calling `hasOwnProperty` directly!

Reply:
**suggestion:** `src/useDCPWorker.tsx:498` contains the remaining uses of `hasOwnProperty` which should be replaced with `Object.hasOwn`.